app/controllers/participate/login.js

```diff
@@ -15,7 +15,10 @@ export default Ember.Controller.extend({
     authenticate(attrs) {
       this.get('session')
         .authenticate('authenticator:jam-jwt', attrs)
-        .then(() => this.transitionToRoute('participate.survey.consent'))
+        .then(() => {
+          this.set('studyId', attrs['password']);
```

Comment:
This seems like a reasonable way to go. Elsewhere we're [setting attributes on the parent participate controller](https://github.com/CenterForOpenScience/isp/blob/develop/app/controllers/participate/login.js#L42), so for consistency it may be better do save `studyId` there. 

Should we be saving `studyId` on the session too?